Add additional supported formats

andynava opened the pull request:
Updating list of supported formats to match options in portal: https://i.stack.imgur.com/BGvIn.png

azurecla commented:
Hi __@lizrice__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@squillace - Could you review the update, please?

squillace commented:
@tysonn @lizrice looks good. Geez; I searched and didn't get the notification. @lizrice, my apologies for the delay. Thanks for the work. 

Tyson, can this be merged still, or do we need to rebase?

andynava committed:
Add additional supported formats

Updating list of supported formats to match options in portal: https://i.stack.imgur.com/BGvIn.png
